texts/en_US.properties

```diff
@@ -18,6 +18,14 @@ geyser.auth.login.offline=Attempting to login using offline mode... authenticati
 geyser.auth.login.wait=Please wait until you are logged in...
 geyser.auth.stored_credentials=Using stored credentials for bedrock user {0}
 
+geyser.bootstrap.args.confignotspecified=Please specify config file with -c option.
```

Comment:
Maybe make `-c` a parameter of the string? So people don't accidentally translate it.